index.html

```diff
@@ -908,10 +908,10 @@ <h2>Extensions to the `Navigator` interface</h2>
         </div>
     </section>
     <section>
-        <h1>Declaring candidate regions for default touch behaviors</h1>
-        <p>For touch input, the default action of any and all pointer events MUST NOT be a manipulation of the viewport (e.g. panning or zooming).</p>
-        <div class="note">Touch manipulations are intentionally not a default action of pointer events. Removing this dependency on the cancellation of events facilitates performance optimizations by the user agent.</div>
-        <div class="note">While the issue of pointers used to manipulate the viewport is generally limited to touch input (where a user's finger can both interact with content and pan/zoom the page), certain user agents may also allow the same types of (direct or indirect) manipulation for other pointer types. For instance, on mobile/tablet devices, users may also be able to pan using a stylus. This section applies to these scenarios as well (despite the specification's use of "touch").</div>
+        <h1>Declaring candidate regions for default direct manipulation behaviors</h1>
+        <p>For <a>direct manipulation</a> interactions, the default action of any and all pointer events MUST NOT be a manipulation of the viewport (e.g. scrolling or zooming).</p>
```

Comment:
is it panning or scrolling?

Reply:
we use(d) the terms interchangeably, but yes we should stick to one ... think when/if we merge the other PR for the new section (which messes things around more fundamentally) and rebase/merge it into this, we can revisit that. my preference is on panning